toolkit/locales/en-US/chrome/global/aboutSupport.ftl

```diff
@@ -45,6 +65,13 @@ aboutSupport-appBasicsMultiProcessSupport = Multiprocess Windows
 aboutSupport-appBasicsSafeMode = Safe Mode
 aboutSupport-appBasicsProfiles = Profiles
 
+# This string displays the number of open multiprocess (e10s) windows compared
+# to the overall number of open windows, and the current status of e10s in the
+# browser.
+# $remote (Number) - Number of multiprocess windows
+# $total (Number)  - Total number of open windows
+# $status (String) - Current status of e10s
```

Comment:
I wrote `{ $remote }/{ $total } ({ $status })` for the sake of explaining the comment, I don't expect localizers to change it.

Having said that, the goal of a localization comment is to explain what the final result will be to a localizer, and I think saying that `$status` is a number would only make it confusing, because what localizers care about is that a string will be displayed between parenthesis, not the number. If that's needed for semantic comments to work, that's a different story.

And the more I think about this comment, the more I realize how this might be broken from a localizer's point of view, while it's technically clean and simple: how are you going to display this kind of string for localization? Are they going to see individual status messages or the whole syntax blurb?